### PR TITLE
Updating mintlist-cli to 0.1.0

### PR DESCRIPTION
**Pull request by terranmoccasin**
Updating `mintlist-cli` for new mintlist management in https://github.com/orca-so/orca-mintlists.

- Remove tokenlist generation
- Remove index file generation
- Add linting and formatting commands

### CODE REVIEW

File: packages/mintlist-cli/src/util/mintlist-file-util.ts
Comment (yugure-orca) [inline packages/mintlist-cli/src/util/mintlist-file-util.ts:1]: nit: Could we use ``MintlistFileUtil.cmpMint`` where ``localeCompare`` is used ?

**Review by yugure-orca — COMMENTED** [review]
Nice updates!

IIRC, nathan asking if field name is "image" or "url".
It would be nice to be able to check for logical errors. 🙏

File: packages/mintlist-cli/src/util/mintlist-file-util.ts
Comment (terranmoccasin) [reply packages/mintlist-cli/src/util/mintlist-file-util.ts:1]: Ah, nice catch!

**Review by yugure-orca — APPROVED** [review]
Thanks for the updates!
LGTM!

**Pull request merged**